SwissTransfer/Sources/SwissTransferApp.swift

```diff
@@ -87,6 +92,33 @@ struct SwissTransferApp: App {
             }
 
             try await currentManager.tryUpdatingAllTransfers()
+
+            checkAppVersion()
```

Comment:
You don't have to be in a Task to call the method, you can call it before.
In this case, we have to finish `tryUpdatingAllTransfers` before checking the version